--- log-gearman-client.py	2016-07-11 13:43:06.158197457 +0000
+++ /root/log-gearman-client.py	2016-07-07 14:50:47.846197457 +0000
@@ -97,8 +97,9 @@
                 logging.exception("Exception submitting job to Gearman.")
 
     def _get_log_dir(self, event):
-        parameters = event["build"].get("parameters", {})
-        base = parameters.get('LOG_PATH', 'UNKNOWN')
+        #parameters = event["build"].get("parameters", {})
+        #base = parameters.get('LOG_PATH', 'UNKNOWN')
+        base = event["build"].get("url", 'UKNOWN')
         return base
 
     def _parse_fields(self, event, filename):
@@ -168,7 +169,7 @@
     def setup_processors(self):
         for publisher in self.config['zmq-publishers']:
             gearclient = gear.Client()
-            gearclient.addServer('localhost')
+            gearclient.addServer('localhost', port=4731)
             gearclient.waitForServer()
             log_processor = EventProcessor(
                 publisher, gearclient,
@@ -199,6 +200,7 @@
         if statsd_host:
             self.wait_for_name_resolution(statsd_host, statsd_port)
         self.gearserver = gear.Server(
+            port=4731,
             statsd_host=statsd_host,
             statsd_port=statsd_port,
             statsd_prefix=statsd_prefix)
